foreshadow/steps/preparerstep.py

```diff
@@ -542,6 +542,11 @@ def check_process(self, X):
         )
         self._parallel_process = self.parallelize_smart_steps(X)
 
+    def _fit_transform(self, X, y=None, **fit_params):
+        if isinstance(self._parallel_process, ParallelProcessor):
+            self._parallel_process.configure_step_name(self.__class__.__name__)
```

Comment:
Is this necessary? Shouldn't the default value of the step_name already set when constructing the preparerstep instance?